config/nav.yml

```diff
@@ -304,5 +304,4 @@ nav:
         - deepc: about/case-studies/deepc.md
         - Outfit7: about/case-studies/outfit7.md
         - Puppet: about/case-studies/puppet.md
-        - PNC Bank: about/case-studies/pnc.md
-    - Blog: /blog/
```

Comment:
Why does this remove the `/blog` portion of the nav here?

Reply:
The file I had locally did not have it. so I think it got removed while I was merging conflict. I started working on a brand new branch and still things went haywire. I need help :(